getting-started/setup-multi-node-auth.md

```diff
@@ -38,26 +131,25 @@ Note that any previously created user passwords will need to be recreated to
 incorporate the new encryption.
 
 ##### Enable password authentication on the data nodes
-As with the [basic setup][], authentication is set by modifying the HBA file (by default
-`pg_hba.conf`, in the data directory). Add a line to `pg_hba.conf` to enable encrypted 
-authentication between the access node and the data node for all users:
+As with the [trust authentication](#multi-node-auth-trust), password authentication is
```

Comment:
I suggest to remove reference to trust authentication. IMHO it is unlikely the reader will read the trust authentication first and then the password authentication.
```suggestion
Password authentication is
```

Reply:
I guess it is necessary to repeat how to find the hba file.

Reply:
Removed reference and added a note about using `SHOW hba_file`.